### ISSUE TITLE
Is this project alive?

### ISSUE THREAD

**Issue by jonisak** [Slyce-Inc/SlyceMessaging#51]
Hi,

I'm wondering if this project is alive? I want to know if It's good to use this in a new project?

Best regards
jonisak

**Comment by xero88** [comment]
I think it's dead... I'm searching for some alternatives too

**Comment by adriamt** [comment]
Do you found some alternative? I'm currently using this project but I need to do some changes everytime

**Comment by felixdivo** [comment]
I think it's pretty much dead und full of unresolved bugs. :'(

Here is some good looking laternative: https://github.com/stfalcon-studio/ChatKit
But I have not used it yet

**Comment by atroutt** [comment]
Yeah, I'm not at Slyce anymore and I don't know that anyone is working on this as part of their job currently. There was talk of making this the foundation of the Android version of JSQMessagesViewController https://github.com/MessageKit/MessageKit-Android but AFAIK that hasn't started yet (possibly because I volunteered to help with that but haven't made time to do so).

**Comment by custanator** [comment]
Use https://github.com/custanator/SlyceMessaging
